core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/externaldatabase/ExternalDatabaseService.java

```diff
@@ -99,13 +108,21 @@ public void stopDatabase(Cluster cluster, DatabaseAvailabilityType externalDatab
                 externalDatabase.name(), environment.getName(), cluster.getName());
         String databaseCrn = cluster.getDatabaseServerCrn();
         try {
-            redbeamsClient.stopByCrn(databaseCrn);
-            waitAndGetDatabase(cluster, databaseCrn, DatabaseOperation.STOP, false);
+            if (externalDatabaseReferenceExist(databaseCrn)) {
+                redbeamsClient.stopByCrn(databaseCrn);
+                waitAndGetDatabase(cluster, databaseCrn, DatabaseOperation.STOP, false);
+            } else {
+                LOGGER.warn("[INVESTIGATE] The external database type was {} but there was no crn", externalDatabase);
```

Comment:
Not sure if the MDCcontext is correct at these places, but if not then probably it would be better to include more information in the logs, like data hub name or crn.

Reply:
it must be correct. we checked the logs and the information is there. I just want to highlight how many times it happened.